runtime/tests/tests.rs

```diff
@@ -679,6 +679,7 @@ fn contract_call_fails_with_insufficient_gas_with_fee_exchange() {
 		});
 }
 
+#[ignore]
```

Comment:
We can un-ignore these now and can update the fee amounts in the tests to match.
Would be good to know the reason for the change but not a blocker for this PR

Reply:
done